test/conformance/api/v1/volumes_test.go

```diff
@@ -80,17 +80,8 @@ func TestConfigMapVolume(t *testing.T) {
 		},
 	})
 
-	withOptionalBadVolume := WithVolume("blah", "/does/not/matter", corev1.VolumeSource{
```

Comment:
Why are we removing all these?

Reply:
The secrets included in the bad volumes won't be found, so the service creation will fail. Do you think we need a new e2e/conformance test for creating a service with an invalid secret?

Reply:
Sounds like that wouldn't be a bad idea, yeah. There's also no secret in this one, should we keep it?